packages/react/src/components/DataGrid/index.stories.mdx

```diff
@@ -0,0 +1,214 @@
+a `headerName` property that will be used as the column header.
+
+### Custom Columns
+
+Columns that display custom content must define a `renderCell` property that will be used to render the content of the
+cell. This property is a function that takes a row as argument and must return a ReactNode.
+
+### Highlight
+
+You can highlight any column by setting the `highlight` property to `true`. This will render the column with a bold
+font.
+
+## Sorting
+
+By default sorting is enabled on columns but you can disable it by setting the `enableSorting` property of columns to `false`.
+Bare in mind that with `SimpleDataGrid` this is all you have to do, but with the `DataGrid` component more work is
+needed as you will need to provide `sortModel` and `onSortModelChange` props.
+
+> ⚠️ Even though `sortModel` is an array, only the first element will be used for now.
+
+## Pagination
+
+The pagination for `DataGrid` is exactly the same as [Pagination](?path=/story/components-pagination-doc--page),
+that's why you can see that `usePagination` hook is used to control the state of the pagination. You can see it in action
+in the server side example.
+
+## Row Selection
+
+In order to enable row selection you need to define the following props: `enableRowSelection`, `rowSelection` and
+`onRowSelectionChange` props.
+
+> This feature is controlled even with `SimpleDataGrid` because this is a type of data that you want to keep track of
+outside the grid component ( for example to delete selected rows ).
+
+## Loading
+
+The component provides out of the box a loading state that can be enabled by setting the `isLoading` props to `true`.
+So feel free to use it between page or sorting changes when you are fetching data from a server.
+
+<Canvas withSource="open">
+  <Story id="components-datagrid--loading"/>
+</Canvas>
+
+## Empty State
+
+The component automatically displays an empty state when there is no data to display and it is not loading.
+
+<Canvas withSource="open">
+  <Story id="components-datagrid--empty"/>
+</Canvas>
+
+## Do's and don'ts
+
+### Button
+
+- ❌ Don’t place button(s) in other column than the last column
+- ✅ Do place buttons all on the right because it designates the action to be done on the whole row, and end of read parsing by eyes.
+- ✅ Be consistent, if you add a button on a row, you must do the same on the other rows.
+
+<img src="components/DataGrid/resources/dd_1_dn.svg"/>
+<img src="components/DataGrid/resources/dd_1_d.svg"/>
+
+### Primary Button
+
+- ❌ Don’t show on each row the primary button
+- ✅ Do show The primary button only at the hover.
+- ✅ Do use secondary or tertiary buttons for always displayed button.
+
+<img src="components/DataGrid/resources/dd_2_dn.svg"/>
+<img src="components/DataGrid/resources/dd_2_d.svg"/>
+
+### Actions
+
+- ❌ Don’t have more than one main action in a line. If you have other secondary actions, you have to put them in a menu.
+- ✅ You should only have one icon per line
```

Comment:
You are not consistence in your `do` sentences. Sometime they begin with `Do`, sometime not. IMO you should be consistent by choosing one form or the other and to be honest beginning with `Do` does not provide anything and it's harder to read.
I thought you will change it in this PR or maybe I misunderstood your answer to my previous comments about this part of thie PR.

Reply:
I changed this sentence and others because it sounded weird using "do". I decided to rewrite sentence in the most appropriate form in order to maximise its readability regardless of beginning with "Do" or "Don't", so I don't get what you mean here? Can you provide or rewrite these in the form you have in mind please?